python/xorq/cli.py

```diff
@@ -114,8 +122,92 @@ def run_command(expr_path, output_path=None, output_format="parquet"):
             raise ValueError(f"Unknown output_format: {output_format}")
 
 
+@tracer.start_as_current_span("cli.serve_command")
+def serve_command(
+    build_path, host=None, port=None, duckdb_path=None, prometheus_port=None
+):
+    """
+    Serve a built expression via Flight Server
+
+    Parameters
+    ----------
+    build_path : str
+        Path to the build directory (output of xorq build)
+    host : str
+        Host to bind Flight Server
+    port : int or None
+        Port to bind Flight Server (None for random)
+    duckdb_path : str or None
+        Path to duckdb cache DB file
+    """
+
+    span = trace.get_current_span()
+    params = {
+        "build_path": build_path,
+        "host": host,
+        "port": port,
+    }
+    if duckdb_path is not None:
+        params["duckdb_path"] = duckdb_path
+    span.add_event("serve.params", params)
+
+    expr_path = Path(build_path)
+    expr_hash = expr_path.stem
+
+    logger.info(f"Loading expression '{expr_hash}' from {expr_path}")
+    build_manager = BuildManager(expr_path.parent)
+    if not build_manager.artifact_store.exists(expr_hash, "expr.yaml"):
+        raise ValueError(f"Error: expr.yaml not found in build directory {expr_path}")
+
+    expr = build_manager.load_expr(expr_hash)
+
+    if duckdb_path:
+        db_path = Path(duckdb_path)
+    else:
+        db_path = "xorq_serve.db"  # what should be the default?
```

Comment:
`db_path` must be a `Path`, but the `else` condition is only a `str`
`db_path = Path(duckdb_path or "xorq_serve.db")`